Do not require file or directory for ghost entries (#56).

dranuhl opened the pull request:
This pull request resolves #56.

Done a few tests with RPM 4.11.x, which does not list any as ghost marked files anymore if the not really documented option ``--noghost`` is been given.

ctron commented:
That looks like a simple approach! :+1: 

I would appreciate it however, if you could add a new integration test for this one.

ctron commented on src/it/test16-ghost/pom.xml:
```diff
@@ -0,0 +1,90 @@
+							<outputFileName>test16.rpm</outputFileName>
+
+							<entries>
+								<entry>
```
Should that entry contain `<ghost>true</ghost>`?

dranuhl replied:
Fixed in latest version.

ctron commented:
I updated the master branch to enable GitHub actions also for PRs. And dropped travis. I guess you need to rebase though, if you want to use it.

dranuhl commented:
So I have completed my own development testing for the unit and integration test additions (should have probably switched to draft PR in the meantime, sorry.)

Added 2 changesets:
1. added unit tests for modified PackageEntry class.
2. added integration test for ghost entry handling using native RPM command.

ctron commented:
Looks good to me … do you want me to merge and release?

dranuhl commented:
Sure.

ctron commented:
Release as part of 1.7.0.

Many thanks!
